packages/routing-utils/src/schemas.ts

```diff
@@ -116,6 +116,9 @@ export const routesSchema = {
           isInternal: {
             type: 'boolean',
           },
+          matchUnescapedFirst: {
```

Comment:
Should we shorten this to just `unescape: true | undefined`?

Reply:
The longer name makes more sense to me